apb does not appear in "apb list" after "apb push -o"

My environment was setup using this guide: https://github.com/openshift/ansible-service-broker#getting-started-with-the-ansible-service-broker
```
[root@localhost my-test-apb]# oc status
oc v3.7.0+7ed6862
kubernetes v1.7.6+a08f5eeb62
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://172.17.0.1.nip.io:8443
openshift v3.7.0+7ed6862
kubernetes v1.7.6+a08f5eeb62
```
I went through the steps under "Using apb init" in https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/blob/master/docs/getting_started.md. But "apb list" doesn't show dh-my-test-apb. Only the default apbs.
```
[root@localhost my-test-apb]# apb push -o
version: 1.0
name: my-test-apb
description: This is a sample application generated by apb init
bindable: False
async: optional
metadata:
  displayName: my-test
plans:
  - name: default
    description: This default plan deploys my-test-apb
    free: True
    metadata: {}
    parameters: []
Found registry IP at: 172.30.1.1:5000
Building image with the tag: 172.30.1.1:5000/openshift/my-test-apb
Successfully pushed image: 172.30.1.1:5000/openshift/my-test-apb
https://asb-1338-ansible-service-broker.172.17.0.1.nip.io/ansible-service-broker
Successfully bootstrapped Ansible Service Broker
Successfully relisted the Service Catalog
[root@localhost my-test-apb]# apb list
ID                                NAME                   DESCRIPTION                                                                                                                                                                
ca91b61da8476984f18fc13883ae2fdb  dh-etherpad-apb        Note taking web application                                                                                                                                                
ab24ffd54da0aefdea5277e0edce8425  dh-hastebin-apb        This is a sample application generated by apb init                                                                                                                         
9f7da06f179b895a8ee5f9a3ce4af7ef  dh-hello-world-apb     deploys hello-world web application                                                                                                                                        
b43a4272a6efcaaa3e0b9616324f1099  dh-hello-world-db-apb  A sample APB which deploys Hello World Database                                                                                                                            
f4509733ca0636df3d69b6af53260160  dh-jenkins-apb         Jenkins service with optional persistent storage and S2I build                                                                                                             
6df7afbd132c094704b4a8bfd44378c0  dh-manageiq-apb        ManageIQ                                                                                                                                                                   
67042296c7c95e84142f21f58da2ebfe  dh-mariadb-apb         Mariadb apb implementation                                                                                                                                                 
f6c4486b7fb0cdac4b58e193607f7011  dh-mediawiki-apb       Mediawiki apb implementation                                                                                                                                               
ddd528762894b277001df310a126d5ad  dh-mysql-apb           Software Collections MySQL APB                                                                                                                                             
135bd0df0401e2fdd52fd136935014fb  dh-nginx-apb           An open source reverse proxy web server                                                                                                                                    
1dda1477cace09730bd8ed7a6505607e  dh-postgresql-apb      SCL PostgreSQL apb implementation                                                                                                                                          
0e991006d21029e47abe71acc255e807  dh-pyzip-demo-apb      Python Zip Demo APB Implementation                                                                                                                                         
693cb128e68365830c913631300deac0  dh-pyzip-demo-db-apb   Python Zip Demo Database APB Implementation                                                                                                                                
c65fbd4e701cb71d74fd2cc35e14432b  dh-rds-postgres-apb    Managed relational database service with a choice of six popular database engines.  Set up, operate, and scale a relational database in the cloud with just a few clicks.  
ba9c2d4db404ce97111bea80225de968  dh-rocketchat-apb      This APB deploys RocketChat backed by MongoDB                                                                                                                              
eebf92c7670f30007a4b8db3a8166d5c  dh-thelounge-apb       This is a sample application generated by apb init                                                                                                                         
a946a139a9308a59bf642ac52b4ba317  dh-wordpress-ha-apb    High Availability Wordpress APB    
```

@dymurray I didn't try redeploying yet but i ran `apb bootstrap`. That didn't help. However after adding the local registry config i now see an error in the logs of the ansible-service-broker when i push:

```
 [ERROR] - unable to retrieve image names for registry localregistry - User "system:serviceaccount:ansible-service-broker:asb" cannot list images.image.openshift.io at the cluster scope: User "system:serviceaccount:ansible-service-broker:asb" cannot list all images.image.openshift.io in the cluster (get images)
```

For now i'm just pushing my apbs to dockerhub, that works fine. But it would be nice to use the local registry.

--- a/src/apb/engine.py
+++ b/src/apb/engine.py
@@ -693,6 +693,10 @@ def retrieve_test_result(name, namespace):
 def broker_request(broker, service_route, method, **kwargs):
     if broker is None:
         broker = get_asb_route()
+    else:
+        broker = "%s/ansible-service-broker" % broker
+
+    print "Contacting the ansible-service-broker at: %s%s" %(broker, service_route)
 
     if broker is None:
         raise Exception("Could not find route to ansible-service-broker. "
@@ -981,7 +985,7 @@ def cmdrun_push(**kwargs):
             raise
 
     else:
-        response = broker_request(kwargs["broker"], "/ansible-service-broker/v2/apb", "post", data=data_spec,
+        response = broker_request(kwargs["broker"], "/v2/apb", "post", data=data_spec,
                                   verify=kwargs["verify"],
                                   basic_auth_username=kwargs.get("basic_auth_username"),
                                   basic_auth_password=kwargs.get("basic_auth_password"))
@@ -999,9 +1003,9 @@ def cmdrun_push(**kwargs):
 
 def cmdrun_remove(**kwargs):
     if kwargs["all"]:
-        route = "/ansible-service-broker/v2/apb"
+        route = "/v2/apb"
     elif kwargs["id"] is not None:
-        route = "/ansible-service-broker/v2/apb" + kwargs["id"]
+        route = "/v2/apb" + kwargs["id"]
     else:
         raise Exception("No APB ID specified.  Use --id.")
 

--- a/src/apb/engine.py
+++ b/src/apb/engine.py
@@ -981,7 +981,7 @@ def cmdrun_push(**kwargs):
             raise
 
     else:
-        response = broker_request(kwargs["broker"], "/apb/spec", "post", data=data_spec,
+        response = broker_request(kwargs["broker"], "/ansible-service-broker/v2/apb", "post", data=data_spec,
                                   verify=kwargs["verify"],
                                   basic_auth_username=kwargs.get("basic_auth_username"),
                                   basic_auth_password=kwargs.get("basic_auth_password"))
@@ -999,9 +999,9 @@ def cmdrun_push(**kwargs):
 
 def cmdrun_remove(**kwargs):
     if kwargs["all"]:
-        route = "/apb/spec"
+        route = "/ansible-service-broker/v2/apb"
     elif kwargs["id"] is not None:
-        route = "/apb/spec/" + kwargs["id"]
+        route = "/ansible-service-broker/v2/apb" + kwargs["id"]
     else:
         raise Exception("No APB ID specified.  Use --id.")
 

--- a/src/apb/engine.py
+++ b/src/apb/engine.py
@@ -696,7 +696,7 @@ def broker_request(broker, service_route, method, **kwargs):
     else:
         broker = "%s/ansible-service-broker" % broker
 
-    print "Contacting the ansible-service-broker at: %s%s" %(broker, service_route)
+    print("Contacting the ansible-service-broker at: %s%s" % (broker, service_route))
 
     if broker is None:
         raise Exception("Could not find route to ansible-service-broker. "

